src/parser/shared/modules/AbilityTracker.ts

```diff
@@ -85,6 +75,34 @@ class AbilityTracker extends Analyzer {
     }
     return ability;
   }
+
+  /** Convenience method for getting number of times an ability hit (damage or healing) */
+  getAbilityHits(spellId: number, abilityInfo: Ability | null = null) {
+    const ability = this.getAbility(spellId, abilityInfo);
+    return ability.damageHits + ability.healingHits;
+  }
+
+  /** Convenience method for getting the amount of effective damage an ability did */
+  getAbilityDamage(spellId: number, abilityInfo: Ability | null = null) {
+    return this.getAbility(spellId, abilityInfo).damageVal.effective;
+  }
+
+  /** Convenience method for getting an abilities damage per cast */
+  getAbilityDamagePerCast(spellId: number, abilityInfo: Ability | null = null) {
+    const ability = this.getAbility(spellId, abilityInfo);
+    return ability.damageVal.effective / ability.casts || 0;
+  }
+
+  /** Convenience method for getting the amount of effective healing an ability did */
+  getAbilityHealing(spellId: number, abilityInfo: Ability | null = null) {
+    return this.getAbility(spellId, abilityInfo).healingVal.effective;
+  }
+
+  /** Convenience method for getting an abilities damage per cast */
+  getAbilityHealingPerCast(spellId: number, abilityInfo: Ability | null = null) {
+    const ability = this.getAbility(spellId, abilityInfo);
+    return ability.healingVal.effective / ability.casts || 0;
+  }
```

Comment:
nit: the `PerCast` methods should probably mention that they average in the doc comments. i think averaging is the best option for these, but would be good to be explicit in the doc.